pkgs/by-name/co/cortex-tools/package.nix

```
@@ -0,0 +1,78 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  gitUpdater,
+  versionCheckHook,
+  installShellFiles,
+  stdenv,
+  nix-update-script,
+}:
+buildGoModule (finalAttrs: {
+  pname = "cortex-tools";
+  version = "0.11.3";
+
+  src = fetchFromGitHub {
+    owner = "grafana";
+    repo = finalAttrs.pname;
```

```suggestion
    repo = "cortex-tools";
```
This is a pattern we no longer want to do because it is error prone.

---

Thanks for the tip, I added this.